Addition Comparison Nominating vs Joining a Pool (#4136)
w3f/polkadot-wiki: docs/maintain/maintain-guides-how-to-nominate-polkadot.md

pool is earning rewards and nominating the validators that match your preferences. Participating in
pools is more of a set-and-forget action than nominating by yourself. It is worth noting that it is
the pool operator that maintains the list of validators nominated by the pool, and so, in a way, you
are trusting the pool operator to act in your best interests. However, it is advised to check the
validators nominated by the pool from time to time and change the pool if necessary.

|                                         Nominating                                          |                                                     Joining a Pool                                                      |
| :-----------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------: |
|                                 Minimum 10 DOT to nominate.                                 |                                              Minimum 1 DOT to be a member.                                              |

> | Annual return based on past performance is approximately 15% of the bonded funds. | Annual return based on past performance is approximately 15% of the bonded funds. |

Reviewer: It is not recommended to put a number on the annual return for a nominator or a pool in this table, as it depends on many variables (network variables like staking rate, the value of ideal staking rate, nominator preferences etc). Some nominators receive more than 20% returns and some around 10%. It is okay to put numbers like 15% inflation as it is coded in the protocol, but not the staking returns. I would suggest removing this. Just to make sure I'm not being too cautious here, tagging @laboon 

```suggestion
| Annual return based on past performance is approximately 15% of the bonded funds. | Annual return based on past performance is approximately 15% of the bonded funds. |
```

Reviewer: I'm aware we show this number on Staking Dashboard. As this is a comparison table between nominating directly vs joining pools, if there is no difference in rewards earned whether nominating directly or through pools, we may have to just highlight that instead.

Reviewer 2: Yeah, I wouldn't include anything about historical returns. It varies too much and isn't really relevant. People who want to know this information can calculate it themselves elsewhere.

You should also continue using the term "rewards", and do not change it to "return". 

Just a note, inflation is targeted at 10%, not 15%.

Author: Tab row removed

> | Rewards can be compounded automatically or sent to any account. | Rewards must be manually claimed to compound them or send them to another account. |

Reviewer: As far as I know, it is not possible to set a different account for rewards destination for the pool rewards, whereas for the staking rewards you can.

```suggestion
| Rewards can be compounded automatically or sent to any account. | Rewards can be manually claimed to the pool member's account and be bonded in the pool again to compound them |
```

> | If the active validator gets slashed, all active nominators are subjected to slashing, also those that do not receive rewards. | If the active validator gets slashed, all pool members are subjected to slashing. |

Reviewer: ```suggestion
| If the active validator gets slashed, all active nominators are subjected to slashing, also those that do not receive rewards due to the oversubscription issue. | If the active validator gets slashed, all pool members are subjected to slashing. |
```

|                                Unbonding period of 28 days. Can switch validators without unbonding.                                 |                                              Unbonding period of 28 days. Need to unbond before switching to a different pool.                                              |
|                                      Maximum uncapped.                                      |                                                    Maximum uncapped.                                                    |
|                               Should bond more than the [minimum active nomination](../learn/learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards) in an era to be eligible to earn staking rewards, although it can depend on multiple other factors outlined in the linked document.                               |                                               A nomination pool earns rewards in an era if it satisfies all the conditions mentioned for the nominator (as the nomination pool is just a nominator from [the NPoS system](../learn/learn-phragmen.md) perspective).                                                |

> |                      Nominate up to 16 validators.                      |                                                     Nomination performed by the pool operator.                                                      |

Reviewer: Redundant line. Adding info to the last line instead. Replacing it with Governance info
```suggestion
|                      Staked tokens can be used for participation in Governance                     |        Staked tokens cannot be used for participation in Governance                                                    |
```

| [Rewards payout](../learn/learn-staking-advanced.md#claiming-rewards) can be triggered permissionlessly by anyone (typically done by the validator).   |                                       Rewards must be claimed by the pool member.                                       |
|                            Bonded funds remain in your account.                             | Bonded funds are transferred to a pool account which is administered by the network protocol and is not accessible to anyone else. |

> |                              Nominator is responsible for managing the list of staked validators.                              |                                            Nominations managed by the pool operator.                                            |

Reviewer: ```suggestion
|                              Nominator is responsible for managing the list of staked validators (up to 16).                              |                                            Nominations managed by the pool operator.                                            |
```

stash account, set one up. Note that setting an exchange address as the
recipient of the staking rewards is extremely unsafe.

## Nominating vs Joining a Pool

> Nominating is the action of choosing validators. It does not simply involve bonding tokens. Nominating implies that you constantly monitor your validators to ensure you are always getting your staking rewards but more importantly to make sure you always have genuine validators and decrease the chance of getting slashed. In order to nominate you need a minimum of 10 DOT, and to receive rewards you need at least a balance greater than the minimum active bond. Depending on your validators, if your active validator is oversubscribed you will earn rewards only if your stake is within that of the top 256 nominators.

Reviewer: The values 10 DOT and 256 need to be fetched from on-chain. When @alfarok implements the min active bond react component, that can also be added here. May be add markdown links to nominating and slashing pages

```suggestion
Nominating is the action of choosing validators. It does not simply involve bonding tokens. Nominating is an active task, which implies that you regularly monitor that your stake is backing an active validator in all the eras and check if you are receiving your staking rewards. More importantly, ensure that the validators you chose always act in the best interests of the network protocol and have less chance of getting slashed. To nominate, you need a minimum of 10 DOT, and to receive rewards, you need at least a balance greater than the minimum active bond. Depending on your validators, if your active validator is oversubscribed, you will earn rewards only if your stake is within that of the top 256 nominators. If the validator misbehaves, It is worth noting that your stake is subject to slashing, irrespective of whether you are in the top 256 nominators or not.
```

> Let's be straight, if you want to be eligible to earn rewards while nominating you need 1.5-2 times the minimum active bond. If you cannot make sure you have that number of bonded tokens not only you will not always earn rewards, but you will always be eligible for slashes if you have active validators and those validators get slashed.

Reviewer: The tone of the Wiki should always be technical, reducing opinionated content as much as possible.

```suggestion
As the minimum active bond is a dynamic value, it can make your nomination inactive when the threshold goes above your bonded balance. Hence, to be eligible to earn rewards while nominating, you would need to stake a much higher balance than the minimum active bond.
```

> Nomination pools are a way to participate to staking and earn rewards starting from 1 DOT. The pools are different from custodial solutions because they are native of Polkadot, they are fully decentralized and they are a non-custodial solution. Pool members do not have to choose validators and thus do not have to constantly monitor validators' performance. So, in a way, pools are more a set-and-forget action than being a nominator. The pool creator will do the nominator for you in exchange of a fee.

Reviewer: > The pool creator will do the nominator for you in exchange for a fee.

Could you check if this is the case on the protocol? As far as I know, there is no extra fee you pay the pool creator/operator. what you pay as the transaction fee goes to the network.  

```suggestion
Nomination pools are a way to participate in staking with as little as 1 DOT and earn staking rewards. Nomination pools differ from custodial solutions (like staking through central exchanges) because they are non-custodial, native to Polkadot's protocol, permissionless, transparent, and run in a decentralized way by the community. Before joining a nomination pool, you must ensure that the pool is earning rewards and nominating the validators that match your preferences. Participating in pools is more of a set-and-forget action than nominating by yourself. It is worth noting that it is the pool operator that maintains the list of validators nominated by the pool, and so, in a way, you are trusting the pool operator to act in your best interests. However, it is advised to check the validators nominated by the pool from time to time and change the pool if necessary.
```

Author: There is currently no commission, they are working on it, though. Commission will go out from claiming rewards and go to a payee account.

Reviewer: Cool! You can link the discussion/proposal here to back the statement

> ### Taxation of Staking Rewards
> 
> Remember that depending on the country of residence, staking rewards are subjected to taxes. If you are nominating the staking rewards are constantly paid out with little control on when you can receive them. Pools work differently as you can claim rewards when you desire. This allows you to optimize your taxes as you can claim rewards during bear markets for example (you are taxed on the token value when you receive the token in your wallet).

Reviewer: Although I acknowledge that this is an important piece of info that can benefit the reader, no tax advice or mention of markets in the Wiki due to legal reasons

```suggestion
```

Nomination pools are a way to participate in staking with as little as 1 DOT and earn staking rewards. Nomination pools differ from custodial solutions (like staking through central exchanges) because they are non-custodial, native to Polkadot's protocol, permissionless, transparent, and run in a decentralized way by the community. Before joining a nomination pool, you must ensure that the pool is earning rewards and nominating the validators that match your preferences. Participating in pools is more of a set-and-forget action than nominating by yourself. It is worth noting that it is the pool operator that maintains the list of validators nominated by the pool, and so, in a way, you are trusting the pool operator to act in your best interests. However, it is advised to check the validators nominated by the pool from time to time and change the pool if necessary.

| Nominating | Joining a Pool|
|:--------:|:----------:|
| Minimum 10 DOT to nominate. | Minimum 1 DOT to be a member. |

> | Unlimited bonding period. | Unlimited bonding period until the pool exists. |

Reviewer: ```suggestion
| Can bond and stake DOT indefinitely | Can bond and stake DOT until the pool exists. |
```

> | Unbonding period of 28 days. | Unbonding period of 28 days. |

Reviewer: ```suggestion
| Unbonding period of 28 days. Can switch validators without unbonding | Unbonding period of 28 days. Need to unbond before switching to a different pool |
```

| Maximum uncapped. | Maximum uncapped. |

> | Staking rewards not guaranteed. | Staking rewards guaranteed. |

Reviewer: This is not true. A pool may not earn rewards when none of the validators it nominates make it to the active set
```suggestion
| Should bond more than the [minimum active nomination](../learn/learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards) in an era to be eligible to earn staking rewards, although it can depend on multiple other factors outlined in the linked document. |  A nomination pool earns rewards in an era if it satisfies all the conditions mentioned for the nominator (as the nomination pool is just a nominator from [the NPoS system](../learn/learn-phragmen.md) perspective  |
```

> | Nominate and regularly check up to 16 validators. | No nomination. |

Reviewer: ```suggestion
| Nominate up to 16 validators. | Nomination performed by the pool operator |
```

> | Rewards paid out regularly every 2-3 days by the active validator or any of the nominators. | Rewards must be claimed by the pool member. |

Reviewer: There is a possibility that staking payouts are not triggered by anyone for multiple weeks. Point them to the simple payouts page for further reading.

```suggestion
| [Rewards payout](../learn/learn-staking-advanced.md#claiming-rewards) can be triggered permissionlessly by anyone. (typically done by the validator)  | Rewards must be claimed by the pool member. |
```

> | Bonded funds remain in your account. | Bonded funds are transferred to a pool account not accessible by anyone (only accessible by the pool's internal logic). |

Reviewer: ```suggestion
| Bonded funds remain in your account. | Bonded funds are transferred to a pool account which is administered by the network protocol and is not accessible to anyone else |
```

> | Non-custodial, native to Polkadot | Non-custodial, native to Polkadot |

Reviewer: ```suggestion
| Nominator is responsible for managing the list of staked validators | Pool member plays a passive role and needs to monitor the list of staked validators managed by the pool operator  |
```
